tests/spec/fs.access.spec.js

```diff
@@ -41,8 +43,8 @@ describe('fs.access', function () {
   });
 
   it('should return no error if file does exist and mode = F_OK', function (done) {
-    var fs = util.fs();
-    var contents = 'This is a file.';
+    let fs = util.fs();
```

Comment:
You can follow the same pattern as above, and use `const` here, since `fs` won't change during the scope of this function.

Reply:
I replaced all let statements with const and pushed another commit.